autopush/router/apnsrouter.py

```diff
@@ -129,9 +129,18 @@ def _route(self, notification, router_data):
         }
         if notification.data:
             payload["body"] = notification.data
-            payload["con"] = notification.headers["encoding"]
-            payload["enc"] = notification.headers["encryption"]
-
+            try:
+                payload["con"] = notification.headers["encoding"]
+                if payload["con"] != "aes128gcm":
+                    payload["enc"] = notification.headers["encryption"]
```

Comment:
Checking for "aes128gcm" here might be more accurate but the other routers don't do it: they just add an 'enc' to payload if 'encryption' is in the header. Can we keep it consistent across the routers?

Reply:
Sure. It's probably more consistent to just include the header if it's present since we have other checks in place to see if it's required.